x-pack/metricbeat/module/aws/cloudwatch/cloudwatch.go

```diff
@@ -148,7 +150,7 @@ func (m *MetricSet) Fetch(report mb.ReporterV2) error {
 	// Create events based on listMetricDetailTotal from configuration
 	if len(listMetricDetailTotal.metricsWithStats) != 0 {
 		for _, regionName := range m.MetricSet.RegionsList {
-			//m.logger.Debugf("Collecting metrics from AWS region %s", regionName)
```

Comment:
any idea why this was commented?

Reply:
no clue 😬 I don't think it's intentional.